src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_using_external_timeout_manager.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace NServiceBus.AcceptanceTests.DelayedDelivery
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using Features;
+    using NServiceBus.Config;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_using_external_timeout_manager : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_delay_delivery()
+        {
+            await Scenario.Define<Context>()
+                .WithEndpoint<EndpointWithTimeoutManager>()
+                .WithEndpoint<Endpoint>(b => b.When((session, c) =>
+                {
+                    var options = new SendOptions();
+
+                    options.DelayDeliveryWith(TimeSpan.FromMilliseconds(5000));
```

Comment:
now that the header is checked, I'd say that's giving us enough confidence that the message went through the timeout manager, so I think delaying the test by 5 sec. is overkill. Let's reduce that down to maybe 1 sec?